typescript/src/offerings.ts

```diff
@@ -135,12 +139,44 @@ export enum PRODUCT_CATEGORY {
      * A type of product for non-subscription.
      */
     NON_SUBSCRIPTION = "NON_SUBSCRIPTION",
-  
+
     /**
      * A type of product for subscriptions.
      */
     SUBSCRIPTION = "SUBSCRIPTION",
 
+    /**
+     * A type of product for unknowns.
+     */
+    UNKNOWN = "UNKNOWN",
+}
+
+export enum PRODUCT_TYPE {
+    /**
+     * A consumable in-app purchase.
+     */
+    CONSUMABLE = "CONSUMABLE",
+
+    /**
+     * A non-consumable in-app purchase.
+     */
+    NON_CONSUMABLE = "NON_CONSUMABLE",
+
+    /**
+     * A non-renewing subscription.
+     */
+    NON_RENEWABLE_SUBSCRIPTION = "NON_RENEWABLE_SUBSCRIPTION",
+
+    /**
+     * An auto-renewable subscription.
+     */
+    AUTO_RENEWABLE_SUBSCRIPTION = "AUTO_RENEWABLE_SUBSCRIPTION",
+
+    /**
+     * A subscription that is pre-paid.
```

Comment:
Correct me if I'm wrong but are some of these iOS/Android only? If so, should we add it to the documentation?

Reply:
That's correct, I will add it to the docs